retroarch.c

```diff
@@ -16067,10 +16032,13 @@ int16_t input_joypad_analog(const input_device_driver_t *drv,
             : bind->joyaxis;
 
          /* Analog button. */
-         /* no deadzone/sensitivity correction for analog buttons currently */
          if (drv->axis)
-            res = abs(drv->axis(joypad_info.joy_idx, axis));
-
+         {
+            float normal_mag = 0.0f;
+            if (configuration_settings->floats.input_analog_deadzone)
+               normal_mag = abs((1.0f / 0x7fff) * drv->axis(joypad_info.joy_idx, axis));
```

Comment:
This was already merged, but should this be `fabs` instead? Getting warnings in compiles from it.

Reply:
Yes this was already spotted and fixed here https://github.com/libretro/RetroArch/commit/79ce99715697aafdc88a422039eb246a27484ce4